src/kbmod/standardizers/standardizer.py

```diff
@@ -0,0 +1,589 @@
+"""`Standardizer` converts data from a given data source into a `LayeredImage`
+object, applying data-source specific, transformations in the process. A
+layered image is a collection containing:
+ * science image
+ * variance image
+ * mask image
+along with the:
+ * on-sky coordinates of the pixel in the center of the science image
+ * observation timestamp
+ * location of the data source
+
+When possible, standardizers should attempt to extract a valid WCS and/or
+bounding box from the data source.
+"""
+import os
+import abc
+import glob
+import warnings
+
+
+__all__ = ["Standardizer", "StandardizerConfig", "ConfigurationError"]
+
+
+class ConfigurationError(Exception):
+    """Error that is raised when configuration parameters contain a logical error."""
+    pass
+
+
+class StandardizerConfig:
+    """Standardizer configuration.
```

Comment:
It might be worth pointing out that this is a base class for standardizer configs. The next line implies it.